datasync/downloader/queue.go

```diff
@@ -326,6 +363,15 @@ func (q *queue) Schedule(headers []*types.Header, from uint64) []*types.Header {
 				q.receiptTaskQueue.Push(header, -int64(header.Number.Uint64()))
 			}
 		}
+
+		if q.mode == FastSync && params.IsStakingUpdateInterval(header.Number.Uint64()) {
```

Comment:
It seems I don't know well about this, but I wonder that it is okay that we test the fixed staking interval as `4` in the previous downloader test.

Reply:
According to *fixed* staking interval, AFAIK the two interval (staking update interval, proposer update interval) cannot be updated using governance vote.